pulsar/consumer_impl.go

```diff
@@ -319,6 +302,15 @@ func (c *consumer) Close() {
 	})
 }
 
+func (c *consumer) Seek(msgID MessageID) error {
+	mid, ok := c.messageID(msgID)
+	if !ok {
+		return nil
```

Comment:
Shouldn't we return an error here?

Reply:
I checked other interfaces like Ack and NackID, they didn't handle error, because their interface didn't need return error, but I am not sure the correct behavior here. so I follow the current way.